utils-intervalrdd/src/main/scala/org/bdgenomics/utils/interval/rdd/IntervalPartition.scala

```diff
@@ -103,7 +103,7 @@ protected class IntervalPartition[K <: Interval[K], V: ClassTag](protected val a
 private[rdd] object IntervalPartition {
 
   def apply[K <: Interval[K], K2 <: Interval[K2], V: ClassTag](iter: Iterable[(K, V)]): IntervalPartition[K, V] = {
-    val array = iter.toArray
+    val array = iter.toArray.sortBy(_._1)
```

Comment:
I was getting a bunch of classtag issues when I did it that way. Ill try to find out why.